site/content/introduction/01-Getting-started.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Getting started"
+weight: 10
+---
+
+## Getting started
+
+_Need help? Join [Weave Community Slack][slackjoin]._
+
+To list the details about a cluster or all of the clusters, use:
+
+```
+
+eksctl get cluster [--name=<name>][--region=<region>]
+
+```
+
+To create the same kind of basic cluster, but with a different name, run:
+
+```
+
+eksctl create cluster --name=cluster-1 --nodes=4
+
+```
+
+EKS supports versions `1.10`, `1.11` and `1.12` (default), with `eksctl` you can deploy either version by passing `--version`.
+
+```
+
+eksctl create cluster --version=1.10
+
+```
+
+A default [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) (gp2 volume type provisioned by EBS) will be added automatically when creating a cluster. If you want to prevent this, use the `--storage-class` flag. For example:
+
+```
+
+eksctl create cluster --storage-class=false
+
+```
+
+To write cluster credentials to a file other than default, run:
+
+```
+
+eksctl create cluster --name=cluster-2 --nodes=4 --kubeconfig=./kubeconfig.cluster-2.yaml
+
+```
+
+To prevent storing cluster credentials locally, run:
+
+```
+
+eksctl create cluster --name=cluster-3 --nodes=4 --write-kubeconfig=false
+
+```
+
+To let `eksctl` manage cluster credentials under `~/.kube/eksctl/clusters` directory, run:
+
+```
+
+eksctl create cluster --name=cluster-3 --nodes=4 --auto-kubeconfig
+
+```
+
+To obtain cluster credentials at any point in time, run:
+
+```
+
+eksctl utils write-kubeconfig --name=<name> [--kubeconfig=<path>][--set-kubeconfig-context=<bool>]
+
+```
+
+To use a 3-5 node Auto Scaling Group, run:
+
+```
+
+eksctl create cluster --name=cluster-5 --nodes-min=3 --nodes-max=5
+
+```
+
+> NOTE: You will still need to install and configure autoscaling. See the "Enable Autoscaling" section below. Also note that depending on your workloads you might need to use a separate nodegroup for each AZ. See [Zone-aware Autoscaling](#zone-aware-autoscaling) below for more info.
```

Comment:
Minor nit: we're using "Auto Scaling" (two words) above but "Autoscaling" here. AWS docs use the former: "Auto Scaling".

Reply:
Good catch. It's important for us to be consistent (and consistent with Amazon)!